core/chains/evm/txmgr/orm.go

```diff
@@ -42,6 +42,78 @@ type orm struct {
 
 var _ ORM = (*orm)(nil)
 
+// Structs saved in database should be internal to
+// the ORM and distinct from generic params
+// so that modifying generic params does neccessitate
+// modification of underlying column types
+// (or runtime errors by mistake)
+type dbReceipt struct {
+	ID               int64
+	TxHash           common.Hash
+	BlockHash        common.Hash
+	BlockNumber      int64
+	TransactionIndex uint
+	Receipt          evmtypes.Receipt
+	CreatedAt        time.Time
+}
+
+type dbRawReceipt = evmtypes.Receipt
+
+type dbReceiptPlus struct {
+	ID           uuid.UUID        `db:"id"`
+	Receipt      evmtypes.Receipt `db:"receipt"`
+	FailOnRevert bool             `db:"FailOnRevert"`
+}
+
+func fromDBReceipts(rs []dbReceipt) (receipts []EvmReceipt) {
+	for i := 0; i < len(rs); i++ {
+		r := rs[i]
+		receipts = append(receipts, EvmReceipt{
+			ID:               r.ID,
+			TxHash:           r.TxHash,
+			BlockHash:        r.BlockHash,
+			BlockNumber:      r.BlockNumber,
+			TransactionIndex: r.TransactionIndex,
+			Receipt:          r.Receipt,
+			CreatedAt:        r.CreatedAt,
+		})
+	}
+	return
+}
+
+func fromDBReceiptsPlus(rs []dbReceiptPlus) (receipts []txmgrtypes.ReceiptPlus[evmtypes.Receipt]) {
```

Comment:
What is a receiptsPlus?

Reply:
Are you referring to the extra `s` between Receipt and Plus? It's just the plural form of receiptPlus. 

Otherwsie, receiptPlus just represents the tx receipt + some additional information required by the txmgr